.github/actions/dockerize-neon-tests/action.yml

```diff
@@ -0,0 +1,28 @@
+name: "Build image"
+description: "Build and push neon test docker image"
+inputs:
+  image_tag:  # id of input
+    description: 'neon tests image tag'
+    required: true
+  docker_username:
+    description: 'docker hub username'
+    required: true
+  docker_password:
+    description: 'docker hub password'
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Build and push neon tests docker image
+      id: docker_pipeline
+      shell: bash
+      run: |
+        image_id="neonlabsorg/neon_tests"
+        delimeter=$(printf "%0.s-" {1..30})
```

Comment:
what delimeter did?

Reply:
I thought it is your code :-)

Delimeter makes beautiful 
![image](https://user-images.githubusercontent.com/110330090/222421402-0d8aecfb-5706-4385-953b-74eb887a2e43.png)

Reply:
I thought it is your code :-)

Delimeter makes beautiful

![image](https://user-images.githubusercontent.com/112946046/222422134-56e7dbf3-bbf7-4354-a6fc-5550110e1127.png)